keys/generate

```diff
@@ -1,16 +1,22 @@
 #!/usr/bin/env bash
 
+_MSYSTEM="$MSYSTEM"
 set -e -u
 
 cd $(dirname $0)
 
-docker run --rm -v $PWD/web:/keys concourse/concourse \
+WORK_DIR="$PWD"
```

Comment:
I was reminded of this syntax to provide a default value to a bash variable if it's not initialized
https://bash.cyberciti.biz/guide/Default_shell_variables_value
```suggestion
WORK_DIR="$PWD"
_MSYSTEM="${MSYSTEM:=}"
```

I think this plays nicer with `set -u` instead of avoiding it all together.